component/pom.xml

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c)  2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
```

Comment:
Fix the license. Don't use apache license (use the GPL license). Check other places as well.

Reply:
Updated license in [#045ea86c0689a8343f00de3b61249053414144d0](https://github.com/wso2-extensions/siddhi-gpl-execution-streamingml/pull/16/commits/045ea86c0689a8343f00de3b61249053414144d0)